src/player.rs

```diff
@@ -180,12 +180,12 @@ fn find_available_alternative<'a>(session: &Session, track: &'a Track) -> Option
 fn load_track(session: &Session, track_id: SpotifyId) -> Option<vorbis::Decoder<Subfile<AudioDecrypt<Box<ReadSeek>>>>> {
     let track = session.metadata::<Track>(track_id).await().unwrap();
 
-    info!("Loading track {:?}", track.name);
+    info!("Loading track {}", track.name);
```

Comment:
Keeping quotes around the name would be good.
That's the reason I was using debug formatting, although I agree it's not great for non ASCII names.